common/src/stack/command/stack/commands/__init__.py

```diff
@@ -1893,49 +1893,49 @@ def endOutput(self, header=[], padChar='-', trimOwner=False, trimHeader=False):
 				for i in range(0, rows):
 					header.append('col-%d' % i)
 
-			list = []
+			output_lines = []
 			for line in self.output:
-				dict = {}
+				dictionary = {}
 				for i in range(0, len(header)):
 					if header[i]:
 						key = header[i]
 						val = line[i]
 
-						if key in dict:
-							if not isinstance(dict[key], list):
-								dict[key] = [dict[key]]
-							dict[key].append(val)
+						if key in dictionary:
+							if not isinstance(dictionary[key], output_lines):
```

Comment:
Maybe we could throw away some code here by using DefaultDict instead?

Reply:
Also, arg2 of isinstance() needs to be a type or tuple of types, so `list` in the deleted code was actually the correct usage.